server/api/api.go

```diff
@@ -1322,27 +1323,29 @@ func (a *API) errorResponseWithCode(w http.ResponseWriter, statusCode int, error
 		data = []byte("{}")
 	}
 	w.WriteHeader(statusCode)
-	w.Write(data)
+	_, _ = w.Write(data)
 }
 
 func (a *API) noContainerErrorResponse(w http.ResponseWriter, sourceError error) {
-	a.errorResponseWithCode(w, http.StatusBadRequest, ERROR_NO_WORKSPACE_CODE, ERROR_NO_WORKSPACE_MESSAGE, sourceError)
+	a.errorResponseWithCode(w, http.StatusBadRequest, ErrorNoWorkspaceCode, ErrorNoWorkspaceMessage, sourceError)
 }
 
-func jsonStringResponse(w http.ResponseWriter, code int, message string) {
+func jsonStringResponse(w http.ResponseWriter, code int, message string) { //nolint:unparam
 	w.Header().Set("Content-Type", "application/json")
 	w.WriteHeader(code)
 	fmt.Fprint(w, message)
 }
 
-func jsonBytesResponse(w http.ResponseWriter, code int, json []byte) {
+func jsonBytesResponse(w http.ResponseWriter, code int, json []byte) { //nolint:unparam
```

Comment:
why is needed the `unparam` here? looks like all the 3 parameters are used.

Reply:
Same as below.